zombienet/tests/0007-prepare-warp-sync-db-snapshot.md

```diff
@@ -0,0 +1,39 @@
+# Database snapshot guide
+
+For this guide we will be taking a snapshot of a parachain and relay chain. Please note we are using a local chain here `rococo_local_testnet` and `local_testnet`. Live chains will have different values
+
+*Please ensure that the database is not in current use, i.e no nodes are writing to it*
+
+# How to prepare database for a relaychain
+To prepare snapshot for a relay chain we need to copy the database. 
+
+```
+mkdir -p relaychain-snapshot/alice/data/chains/rococo_local_testnet/db/
+
+cp -r chain-data/alice/data/chains/rococo_local_testnet/db/ relaychain-snapshot/alice/data/chains/rococo_local_testnet/db/ 
```

Comment:
This is a bit tricky, `cp` behaves differently on macos and linux.
On linux you will get a folder structure that has the db folder twice (like `.../db/db/...`). To fix this, you can just change the command to include a trailing `.`.

Same for the other occurrence of course.
```suggestion
cp -r chain-data/alice/data/chains/rococo_local_testnet/db/. relaychain-snapshot/alice/data/chains/rococo_local_testnet/db/ 
```